.github/workflows/documentation-update.yml

```diff
@@ -170,6 +170,13 @@ jobs:
           name: JEST_appeals-service-api-coverage
           path: ${{ env.APPEALS_SERVICE_API }}/coverage
 
+      - name: Archive complexity
```

Comment:
Has this been run in GitHub Actions yet? If not, suggest doing a `chore: to remove` commit where you add this branch into the config, check it runs, then remove it using `git rebase -i HEAD~2`

Reply:
i've done this as a dry run and published to github pages